packages/prettier-config/.prettierrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  printWidth: 80,
```

Comment:
we are extending `eslint-config-airbnb-base` for eslint which has a max-len of 100

Reply:
Done.